calculated_policies/aws_s3_bucket_lower_case_tags/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS S3 / Set default tags on buckets with dynamic metadata.
+
+## User Story
+An organization will often run automation based off specific tags on a resource, such as an S3 bucket. However, this operation can be case sensitive and requires all tags keys and associated values to be all lower case.
+
+## Implementation Details
+Provides a Terraform configuration for creating a smart folder and applying a calculated policy on the `AWS > S3 > Bucket > Tags > Template`.  The Calculated policy will injest all tags across any bucket that has this policy applied. 
+Each tag will be checked for case, and if there is an upper case character, it will be converted to lower case.
+
+### Template Input (GraphQL)
+The template input to a calculated policy is a GraphQL query.  In this case the query selects tagging metadata about the bucket:
+```graphql
+{
+  bucket{
+    turbot {
+      tags
+    }
+  }
+}
+```
+### Template (Nunjucks)
+The template itself is a [Nunjucks formatted template](https://mozilla.github.io/nunjucks/templating.html) with custom logic:
+```
+{%- set keys = [] %}
+{%- for k,v in $.bucket.turbot.tags %}
+  {%- set ignore = keys.push(k) %}
+{%- endfor %}
+{%- set ignore = keys.sort() %}
+{%- set set_keys = [] %}
+{%- for k in keys %}
```

Comment:
`{%- for k in keys %}` can be changed to `{%- for k in keys | sort %}`
Removes line 28